external-crates/move/crates/bytecode-interpreter-testsuite/Cargo.toml

```diff
@@ -1,26 +0,0 @@
-[package]
-name = "bytecode-interpreter-testsuite"
```

Comment:
Does this testsuite need to be deleted?

Reply:
I was able to bring it back (except for certain tests). I got a bit excited :)